tests/modules/accordion.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { Accordion, AccordionProps, Text } from "../../src";
+import { DEFAULT_TEXT, DEFAULT_TITLE } from "../common";
+
+const renderComponent = (
```

Comment:
There are a couple of helper functions here. I tend to prefer the unit tests spec to be as high up the file as possible (unless mocking involved). May I suggest to move these helpers down (below the tests) and add a section header instead?